planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -1205,9 +1222,8 @@ bool StartPlannerModule::hasFinishedPullOut() const
 
   // offset to not finish the module before engage
   constexpr double offset = 0.1;
-  const bool has_finished = arclength_current.length - arclength_pull_out_end.length > offset;
 
-  return has_finished;
+  return arclength_current.length - arclength_pull_out_end.length > offset;
```

Comment:
can the parameter added by this PR: https://github.com/autowarefoundation/autoware.universe/pull/7160 be used here instead of offset?

Reply:
you mean using parameter of `turn_signal_remaining_shift_length_threshold` ?

Reply:
Yes

Reply:
`turn_signal_remaining_shift_length_threshold` is tuning param for some usecase but this offset is not tuning parameter, so we can leave them as it is I feel.

Reply:
I though it can be commonized by changing argument of the function but what they are doing is different.
In your PR, judging based on lateral distance. But in this PR, judging based on longitudinal distance.

Reply:
@kyoichi-sugahara Ah you are right, sorry for the confusion.